packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -30,17 +30,17 @@ Explore the Schemas API: [![Run in Postman](https://run.pstmn.io/button.svg)](ht
 
 ## User Schema operations
 
-Each of the operations described here affects the Schema associated with a single [User Type](/docs/reference/api/user-types). The `${typeId}` element in the URL specifies which type. It can be the literal `default` to operate on the Schema of the default User Type, which is created when the org is initialized, or it can be a schema ID.
+Each of the operations described here affects the Schema associated with a single [user type](/docs/reference/api/user-types). The `${typeId}` element in the URL specifies which type. It can be the literal `default` to operate on the Schema of the default user type, which is created when the org is initialized, or it can be a schema ID.
```

Comment:
```suggestion
Each of the operations described here affects the schema that's associated with a single [user type](/docs/reference/api/user-types). The `${typeId}` element in the URL specifies which type. It can be the literal `default` to operate on the schema of the default user type, which is created when the org is initialized, or it can be a schema ID.
```